snorkel/mtl/snorkel_config.py

```diff
@@ -18,7 +18,7 @@
     "checkpointer_config": {  # Note that checkpointer behavior also depends on log_manager_config
         "checkpoint_dir": None,  # Trainer will set this to log_dir if None
         "checkpoint_factor": 1,  # Checkpoint every this many evaluations
-        "checkpoint_metric": "model/train/all/loss:min",
```

Comment:
Hmm.. nervous that this seemed to work for me before. Maybe the defaults were ok?